tests/semantic/bin1/bin1.dl

```diff
@@ -5,15 +5,14 @@
 // - <souffle root>/licenses/SOUFFLE-UPL.txt
 
 // Test allowed range for binary integers
-.decl Binary (n:number)
+.decl Binary (n:unsigned)
 .output Binary
 
 Binary(0b1).
+Binary(0b11111111111111111111111111111111).
 
 #if RAM_DOMAIN_SIZE == 32
-Binary(0b00000000000000000000000000000000).
-Binary(0b11111111111111111111111111111111).
+Binary(0b11111111111111111111111111111111 + 1).
 #elif RAM_DOMAIN_SIZE == 64
-Binary(0b0000000000000000000000000000000000000000000000000000000000000000).
-Binary(0b1111111111111111111111111111111111111111111111111111111111111111).
+Binary(0b1111111111111111111111111111111111111111111111111111111111111111 + 1).
 #endif
```

Comment:
It would be better to remove the 32/64 bit sections, now that we don't need to rely on overflow. The previous line + `Binary(0b0)` is sufficient to test that the max 32 bit binary number is handled (and varying output based on 32/64 is not something we can test like this).
Alternatively, just remove the overflow lines and keep 32/64 tests for `0...0`